### ISSUE TITLE
[Serious] Functionality requires a dragging movement with no alternative

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#617]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87763 |
| **Checkpoint** | 2.5 Pointer Accessible › Dragging Movements (2.5.7.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 2:47 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Functionality requires a dragging movement and does not have a single-pointer, single-touch alternative.
On Files screen, the user is required to drag and drop the files into the package panel.  See Create Job User Experience video at 6:10 and 12:05.

## Recommendation

Rule : 
All functionality that uses a dragging movement for operation (such as sliders, drag-and-drop interfaces) MUST have an alternative that can be achieved by a single pointer without dragging, unless dragging is essential. (A keyboard equivalent does not satisfy this Success Criterion.)

How To Fix: 
Fix this issue by providing an alternative method to accomplish the same function that can be accomplished using single taps such as alternative input into a text field, or a tap to select and a tap to drop.

The alternative method can replace the current method or be an additional method.

Reference: 
The Road to Accessible Drag and Drop: https://vispero.com/resources/the-road-to-accessible-drag-and-drop-part-1/
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements

Background: 
When functionality requires dragging actions, people with motor disabilities may find it difficult or impossible to perform the gesture, either because they may lack the accuracy necessary to carry it out or because they use an assistive technology that lacks the capability or accuracy. An alternative method to accomplish the same function that relies only on single-tap actions that are not path-based must be provided. This allows people with low dexterity or who use assistive technologies to access the functionality, as well as aids people with cognitive disabilities who may not know about or understand the required complex gesture.

 NOTE: Dragging is an operation where the pointer engages with an element on the down event and the element (or a representation of its position) follows the pointer. Examples of dragging actions include dragging to sort or reorder items, dragging to another position on the screen, dragging to draw or change the size of a shape or line. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.

## Screenshots

![03_drag and drop](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87763/03_drag_and_drop.png)
